contacts24/errors.py

```diff
@@ -1,12 +1,16 @@
-HELP_ERROR_MESSAGE = """Supported functions:
-- 'add': Add new contact, the correct format is 'add username phone'
-- 'change': Change existing contact, the correct format is 'change username phone'
-- 'phone': Print existing contact number, the correct format is 'phone username'
-- 'add-birthday': command expects two arguments 'name' and 'birthday'
-- 'show-birthday': Print existing contact birthday, the correct format is 'show-birthday username'
-- 'birthdays': Print birthdays upcoming next week if any, the correct format is 'birthdays'
-- 'all': Print all existing contact numbers if any, the correct format is 'all'
-- 'exit' | 'close': Close the app, the correct format 'exit' and 'close'
+HELP_ERROR_MESSAGE = """\
```

Comment:
I would simplify this. adding new commands to this is time consuming. only format columns without | will work. also, we dont have description for date and phone format which are not obvious

Reply:
I was lazy enough so that it was formated with GPT, but will think about simplifying it.

Reply:
@IhorNi  could we introduce a command as an object that provides its own help/info?